### DISCUSSION TITLE
New Casks Not Showing up in Search

### DISCUSSION THREAD

**Discussion by mattpackwood** [Homebrew/discussions#5320] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.19
ORIGIN: https://github.com/Homebrew/brew
HEAD: 54bea636b41f7953c01a7da5f488f08c1b42ffd1
Last commit: 7 days ago
Core tap JSON: 28 Apr 13:28 UTC
Core cask tap JSON: 28 Apr 13:28 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: nano
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 14.4.1-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: 15.3
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

I see there are new Formulae and Casks when I type `brew update`
And they appear to be be installable
But they are not showing up when I search for them in the website, I know that it can take a short time for them to be added but (for example) descope showed up on 4/21 and still if not showing in the site.

### Relevant casks

limitless, metarename, metavideo, mono-mdk-for-visual-studio

**Comment by SMillerDev** [comment]
I kicked off a new indexing run yesterday since the scheduled ones seemed to have failed for a while. It should be good now
